docs/UserGuide.md

```diff
@@ -562,18 +562,17 @@ In the case where GuestBook rejects your field inputs, you can refer to this sec
-| **`PHONE`**             | `p/`                 | Should contain only **numbers**, and it should be at least 3 digits and at most 15 digits long.                                                                                                                                                                                                                                                                                                  |
+| Field                   | Corresponding prefix | Format/Constraints                                                                                                                                                                                                                                                                                                                                                                              |
+|-------------------------|----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`NAME`**              | `n/`                 | Should contain only **alphabets, spaces, and special characters (`'`, `-`, `.`, `,`, `/`)**.                                                                                                                                                                                                                                                                                                    |
+| **`PHONE`**             | `p/`                 | Should contain only **numbers**, and it should be at least 3 digits and at most 15 digits long.                                                                                                                                                                                                                                                                                                 |
 | **`EMAIL`**             | `e/`                 | Should contain only **alphanumeric characters and 1 `@` character**.<br/>The local segment (before `@`) should be at most 64 characters long, and the domain segment (after `@`) should be at most 255 characters long.<br/>Special characters (`+`, `_`, `.`, `-`) can be included but **not as the first or last characters** of any segment. They also cannot be used adjacent to each other. |
```

Comment:
Should we mention the special characters thing at the first sentence? or move the first sentence to the back?

Reply:
Yes, changed